src/frame_ids.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file is generated. Can you add it to the .gitignore and remove it from your PR